plugins/EpadDB.js

```diff
@@ -2334,23 +2335,19 @@ async function epaddb(fastify, options, done) {
         if (data.dataValues.queueproject !== null) {
           pluginObj.project = { ...data.dataValues.queueproject.dataValues };
         }
-
-        const dock = new DockerService(fs, fastify, path);
-        const containerName = `epadplugin_${pluginObj.id}`;
-        // eslint-disable-next-line no-await-in-loop
-        const resInspect = await dock.checkContainerExistance(containerName);
-
         try {
+          const dock = new DockerService(fs, fastify, path);
+          const containerName = `epadplugin_${pluginObj.id}`;
+          // eslint-disable-next-line no-await-in-loop
+          const resInspect = await dock.checkContainerExistance(containerName);
           if (resInspect.message === '404') {
             fastify.log.info('not a real error. Container has not found so we can create new one');
-            if (sequence) {
-              nonseqresult.push(pluginObj);
-              seqresult.push(pluginObj);
-            } else {
-              throw new Error('404');
+            nonseqresult.push(pluginObj);
+            seqresult.push(pluginObj);
+            if (!sequence) {
+              fastify.runPluginsQueueInternal(nonseqresult, request);
```

Comment:
don't you need await for runPluginsQueueInternal

Reply:
no we don't need await because this section runs in parallel